### ISSUE TITLE
“Docs” header inconsistent with the rest of vanillaframework.io

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-web-and-design/vanillaframework.io#282]
1\. Go to [the “Docs” section of the site](https://docs.vanillaframework.io/).
2\. Go anywhere else on the site.

What you see:

1. ![image](https://user-images.githubusercontent.com/19801137/73607758-2fffa600-45b2-11ea-816d-5313385e13eb.png)

2. ![image](https://user-images.githubusercontent.com/19801137/73607765-3ee65880-45b2-11ea-8c1a-76bcc2de08cc.png)

What you should see:
- The header links are the same across the site
- The header links are in exactly the same position across the site.

This is a regression of #104. I guess that means the header is being maintained in two separate codebases, so that it repeatedly accidentally diverges. Perhaps one could import from the other?

**Comment by bartaz** [comment]
Yes, Vanilla docs and Vanilla 'site' are 2 separate codebases which causes issues like this.
To make things more difficult docs use full width design, while site has grid restricted navigation.
This makes it impossible to align them as they currently are.

@nottrobin was suggesting during docs Flask migration that for consistency with our other products we should consider making docs part of vanilla site (/docs) rather than separate domain. As vanilla site (apart from docs) is simply 5 static pages it shouldn't be hard to maintain it all in one codebase.

The design question would remain of course about the width of the docs site.
Should the docs remain full with (and site be updated to full width as well)? Can the two have 2 separate navigation widths? Or Should both be constrained by grid.

We have similar problems on other sites.
https://jaas.ai/ is a site limited by grid width, but https://jaas.ai/models is separate app (within jass.ai) that has completely different, full width design.

@lyubomir-popov @spencerbygraves Any thoughts on docs vs site layout width for Vanilla?

**Comment by nottrobin** [comment]
@bartaz My point is that if Vanilla considers the full-width layout to be the best way to present docs, why aren't snapcraft.io/docs, jaas.ai/docs, maas.io/docs and ubuntu.com/kubernetes/docs full-width?

My understanding is that these docs are more modern products of our design thinking than docs.vanillaframework.io. And that's definitely not good because Vanilla should lead the way on this stuff, not lag behind. It's Vanilla's job to decide what the right presentation is and then to exemplify it.

So I don't really consider it a "suggestion", I consider it an essential piece of work we haven't got to yet. Is there any disagreement?

(It's fine to take some ideas from docs.vanillaframework.io as enhancements to the other docs platforms, but the first priority is to get Vanilla to be actually presenting our most recent patterns)

**Comment by lyubomir-popov** [comment]
In my opinion the vanilla documentation needs to be aligned with other sites' documentations, meaning fixed width. I actually tried to do it once, as a small task, but it is a bigger piece of work. 

That said, all of the documentations have some cosmetic issues of their own, e.g. not using vanilla framework's grid correctly - in the screenshot below, the sidebar is not part of the  12 column grid, instead the 12 columns are squeezed into the main column:

![image](https://user-images.githubusercontent.com/2741678/74138872-42996100-4bea-11ea-896c-21b96aeba581.png)

So my suggestion would be to update the vanilla doc to properly use the grid, then propagate that change to the rest of the sites.

**Comment by anthonydillon** [comment]
I am waiting on [the Layouts feature](https://github.com/canonical-web-and-design/vanilla-squad/issues/578) to land in Vanilla to apply this change to the docs for Vanilla as the first step to rolling out the updates to the other docs.

**Comment by bartaz** [comment]
@anthonydillon Good point.

I guess it fair to say that we should wait with any significant layout changes until we have layouts implemented into Vanilla, and Vanilla docs can be first docs site to use those.

With that in mind docs vs site navigation will remain inconsistent in terms of layout, but we should at least make sure they both have the same navigation items.

**Comment by bartaz** [comment]
Marking it as blocked for now.
We want to wait for layouts feature in Vanilla and/or the merge of Vanilla site and docs into one project.
